stylix/testbed/default.nix

```diff
@@ -6,26 +6,35 @@
 }:
 
 let
+  availableGraphicalEnvironments = lib.pipe ./graphicalEnvironments [
+    builtins.readDir
+    (lib.mapAttrsToList (name: _: lib.removeSuffix ".nix" name))
+  ];
+
   makeTestbed =
     name: testbed:
     let
 
       system = lib.nixosSystem {
         inherit (pkgs) system;
 
-        modules = [
-          ./modules/common.nix
-          ./modules/enable.nix
-          ./modules/application.nix
-          inputs.self.nixosModules.stylix
-          inputs.home-manager.nixosModules.home-manager
-          testbed
+        modules =
+          [
+            ./modules/common.nix
+            ./modules/enable.nix
+            ./modules/application.nix
+            inputs.self.nixosModules.stylix
+            inputs.home-manager.nixosModules.home-manager
+            testbed
 
-          # modules for external targets
-          inputs.nvf.nixosModules.default
-          inputs.nixvim.nixosModules.nixvim
-          inputs.spicetify-nix.nixosModules.spicetify
-        ];
+            # modules for external targets
+            inputs.nvf.nixosModules.default
+            inputs.nixvim.nixosModules.nixvim
+            inputs.spicetify-nix.nixosModules.spicetify
+          ]
+          ++ map (
+            name: import ./graphicalEnvironments/${name}.nix
```

Comment:
The `/stylix/testbed/graphicalEnvironments/` directory should be renamed to `/stylix/testbed/graphical-environments/`.